source/extensions/filters/common/local_ratelimit/local_ratelimit_impl.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <chrono>
+
+#include "envoy/event/dispatcher.h"
+#include "envoy/event/timer.h"
+
+#include "common/common/thread_synchronizer.h"
+
+#include "extensions/filters/common/local_ratelimit/local_ratelimit.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Filters {
+namespace Common {
+namespace LocalRateLimit {
+
+// TODO(htuch): We should have only one client per thread, but today we create one per filter stack.
+// This will require support for more than one outstanding request per client (limit() assumes only
+// one today).
```

Comment:
Does this TODO belong here? What does it refer to?

Reply:
leftover from copying/pasting from the global rate limiter. Removing.